torch/csrc/cuda/nccl.cpp

```diff
@@ -786,8 +786,8 @@ void all2all_single_equal_split(
     ncclComm_t _comm,
     at::cuda::CUDAStream& stream) {
 #ifdef USE_NCCL
-#if defined(NCCL_MAJOR) && (NCCL_MAJOR == 2) && \
-    (NCCL_MAJOR * 10 + NCCL_MINOR) >= 27
+#if defined(NCCL_MAJOR) && \
+    ((NCCL_MAJOR > 2) || ((NCCL_MAJOR == 2) && (NCCL_MINOR > 7)))
```

Comment:
Is this new condition equivalent to >=27 ? seems to rule out NCCL_MINOR=7.

Reply:
ah good catch! Will fix this